packages/pulp/pulp.spec

```diff
@@ -474,28 +474,30 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_usr}/lib/tmpfiles.d/
 %endif
 # 640 root:apache
-%defattr(640,root,apache,-)
+%defattr(640,apache,pulp,-)
 %ghost %{_sysconfdir}/pki/%{name}/ca.key
 %ghost %{_sysconfdir}/pki/%{name}/ca.crt
 %ghost %{_sysconfdir}/pki/%{name}/rsa.key
 %ghost %{_sysconfdir}/pki/%{name}/rsa_pub.key
 %config(noreplace) %{_sysconfdir}/%{name}/server.conf
 # - apache:apache
-%defattr(-,apache,apache,-)
+%defattr(-,apache,pulp,-)
```

Comment:
@dkliban , would you mind changing comments above those changes to reflect them?

Reply:
don't mind at all